info.xml

```diff
@@ -19,7 +19,7 @@
   <develStage>dev</develStage>
   <compatibility>
     <!-- When set to 5.67 or later class X_EntityFile can be removed -->
```

Comment:
… deleting `X_EntityFile` to be done in a follow-up PR?

Reply:
Yes, will be done in a separate PR.